js/main.js

```diff
@@ -1,2 +1,31 @@
 /* global data */
 /* exported data */
+
+var imageField = document.querySelector('#image-URL');
+var titleField = document.querySelector('#Title');
+var notesField = document.querySelector('#Notes');
+var imageChange = document.querySelector('.image-to-change');
+imageField.addEventListener('input', changeImage);
```

Comment:
Move this below the variables you have declared. In general, all global variables should be listed first at the top of a js document.